docs/how-to/h-manage-app.md

```diff
@@ -51,4 +51,4 @@ juju run-action mysql/leader set-password password=<password> --wait
 
 ```shell
 juju run-action mysql/leader set-password --wait
-```
+```
```

Comment:
This was not checked before, but the for mysql `set-password`  is used to set any of the system users passwords, the default being the `root` user and not `clusteradmin` (i.e the "operator" user) as the text suggests.

To achieve what the text suggest one would run:

```shell
juju run-action mysql/leader set-password username=clusteradmin password=<password> --wait
```